.github/workflows/test_on_push.yaml

```diff
@@ -7,23 +7,15 @@ on:
   pull_request:
 
 jobs:
-  test:
-    name: "Test code and check lint"
+  tests:
+    name: "Tests"
     runs-on: "ubuntu-latest"
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: '3.x'
+          python-version: '3.8'
```

Comment:
Is there a reason to use 3.8?

Reply:
Nope, not really. Will update

Reply:
We should create/document a dev workflow that provides python at the specified version since none of us will have the same system python when deving locally. If we are going to be pinning python in the test/build environment then we should also provide a way for that same python to be used in the local setup.

Reply:
I think pyenv is the best tool for this. I can write a quick walkthrough, but where should that go? In our osd-documentation repo?

Reply:
Added this pull-request to address that. Please let me know what you think. https://github.com/omnivector-solutions/Documentation/pull/12

Reply:
Addressed the original comment in the latest commit